domains/kkll.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "K-K-L-L",
+        "email": "kkll345643@protonmail.com"
+    },
+    "record": {
+        "CNAME": "is-a.dev"
```

Comment:
please put a valid record, this wont work

Reply:
mb i forgot is-a.dev hosting shut down recently

Reply:
but even if is-a.dev hosting was still available it would still be wrong lmfao

Reply:
lol